utils/validations/test_mocks.go

```diff
@@ -111,6 +112,18 @@ func getXscServerApiHandler(t *testing.T, params MockServerParams) func(w http.R
 				}
 			}
 		}
+		if strings.Contains(r.RequestURI, "/xsc/profile_repos") && isXrayAfterXscMigration {
+			assert.Equal(t, http.MethodPost, r.Method)
+			w.WriteHeader(http.StatusOK)
+			content, err := os.ReadFile("../../tests/testdata/other/configProfile/configProfileExample.json")
+			if !assert.NoError(t, err) {
+				return
+			}
+			_, err = w.Write(content)
+			if !assert.NoError(t, err) {
+				return
+			}
+		}
```

Comment:
Move this to `XrayServer` and use it instead of `XscServer` since this API is only available in the new inner service inside Xray Server